app/Http/Controllers/ProductController.php

```diff
@@ -26,7 +26,7 @@ public function __construct()
      */
     public function index()
     {
-        $products = Product::all();
+        $products = Product::paginate(10);
```

Comment:
So cool! So this only loads the first 10 items, maybe add a button to load the next 10 items. But great start kiddo!

Reply:
Yes. But main page is now broken ;-)